codesydney/codesydneywebsite
# Issue 8 gallery

## Rationale

What does this pull request do?

Initial for the gallery page.

## Advice for Reviewers & Testing Notes

- Explain how your change should be reviewed, and where reviewers should direct their attention.

> New files 

1. src/elements/Gallery.js
2. public/assets/scss/elements/_gallery.scss
3. src/elements/common/SubPageTitle.js
4. public/assets/scss/common/_subpagetitle.scss
5. public/assets/images/gallery/*.*

>Things done

1. Initialed the gallery page with Carousel display.
2. Set up the component of subpagetitle for re-use and later-adjustment for following other pages.
3. Add gallery imgs to public folder.
4. react-image-gallery api has been installed and used for carousel. [react-image-gallery api](https://www.npmjs.com/package/react-image-gallery)

## Screenshots:

- Add a screenshot or two of your changes, where applicable.

Some issue with screenshot uploading.....



*Note - This allows the issue to be automatically linked to the PR once the PR is submitted for review*
- Fixes #8 Gallery
